test/integration/apiserver/field_validation_test.go

```diff
@@ -2893,3 +2978,541 @@ func setupCRD(t *testing.T, config *rest.Config, apiGroup string, schemaless boo
 
 	return crd
 }
+
+func BenchmarkFieldValidation(b *testing.B) {
+	defer featuregatetesting.SetFeatureGateDuringTest(b, utilfeature.DefaultFeatureGate, features.ServerSideFieldValidation, true)()
```

Comment:
@Jefftree I can't remember what we said about these, are we keeping them?

Reply:
Yeah let's keep them for now and do the removal in a separate PR